Tests/Fixtures/scheme_test/test_project.yml

```diff
@@ -0,0 +1,5 @@
+name: TestProject
```

Comment:
Where is this new Fixture used? Do we need it?

Reply:
This project file is used to generate test project in https://github.com/yonaskolb/XcodeGen/pull/655/files#diff-c6ba86026b68e8323d7e329491a61492R264-R271